libs/wire-api/test/unit/Test/Data/Swagger/Typed.hs

```diff
@@ -0,0 +1,239 @@
+{-# LANGUAGE DerivingVia #-}
+
+module Test.Data.Swagger.Typed where
+
+import Control.Applicative
+import Control.Lens (Prism', prism')
+import Data.Aeson (FromJSON (..), Result (..), ToJSON (..), Value, fromJSON)
+import Data.Aeson.QQ
+import Data.Swagger.Typed
+import Imports
+import Test.Tasty
+import Test.Tasty.HUnit
+
+tests :: TestTree
+tests =
+  testGroup
```

Comment:
Is there an easy way to add a test which shows that a type with a field "name" with an instance of `ToTypedSchema` also has a swagger schema that contains "name"?

Reply:
Yes, I have added some tests for the swagger schema.